scripts/globals/mobskills/guided_missile.lua

```diff
@@ -21,9 +21,12 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local ftp100 = 2
+    local ftp200 = 2.5
+    local ftp300 = 3
     local numhits = 3
     local accmod = 1
-    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, 1, xi.mobskills.magicalTpBonus.NO_EFFECT, 5, 5, 5)
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, 1, xi.mobskills.magicalTpBonus.NO_EFFECT, ftp100, ftp200, ftp300)
```

Comment:
These FTPs are per hit - the FTPs input should be TOTAL ftp which would put this at 6, 7.5, 9

Reply:
> These FTPs are per hit - the FTPs input should be TOTAL ftp which would put this at 6, 7.5, 9

Ah, ok thanks!